tests/calc_001.phpt

```diff
@@ -12,7 +12,8 @@ use Parle\Token;
 
 $p = new RParser;
 $p->token("INTEGER");
-$p->left("'+' '-' '*' '/'");
+$p->left("'+' '-'");
+$p->left("'*' '/'");
```

Comment:
Was just wondering, perhaps this can be solved through the use of the `precendence` method? I'm not sure it is tested properly today.

Thanks

Reply:
The operators need to use %left and this is the classic way of defining the operators.